resources/views/components/search-blood-form.blade.php

```diff
@@ -26,7 +26,7 @@
                     <select name="wilaya" id="wilayaSelect" class="form-select form-select-lg" required>
                         <option selected hidden style="display:none" value="">{{ __('homePage.wilaya') }}</option>
                         @foreach ($wilayas as $wilaya)
-                            <option value="{{ $wilaya['id'] }}">{{ $wilaya['id'].'. '.$wilaya[$wilayaName] }} </option>
+                            <option value="{{ $wilaya['id'] }}">{{ $wilaya['id'].'. '.$wilaya['name'] }} </option>
```

Comment:
$wilaya['name'] will always return the name in Ascii, not in the correct language, let it be $wilaya['nameName'], that last will return the name in the correct language, please have a look at app/View/Components/SearchBloodForm.php line 322

Reply:
Yes , i'm already aware of that , check out the **Wilaya Model**  ( i created accessor for name attribute )

Reply:
thanks , i didn't notice that